playbook-website/app/javascript/components/PbKitPlayground/PlaygroundHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Icon,
+  SectionSeparator,
+  Button,
+  Flex,
+  FlexItem,
+  Image
+} from 'playbook-ui';
+import playgroundLogo from '../../images/playground-logo.svg'
+
+const PlaygroundHeader = () => {
+  const handleBackNavigation = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault()
+    window.location.href = "https://playbook.powerapp.cloud/"
+  };
+
+  return (
+    <>
+      <Flex orientation="row" justify="between" marginTop="xs">
+        <Flex orientation="row" vertical="stretch">
+          <button
+            onClick={handleBackNavigation}
+            style={{ background: 'none', border: 'none', cursor: 'pointer' }}
+            aria-label="Go Back"
+          >
+            <Icon color="primary" fixedWidth icon="chevron-left" marginLeft="sm" size="2x" />
+          </button>
+
+          <SectionSeparator orientation="vertical" inline="flex-container" />
+
+          <Image marginLeft="xs" marginBottom="none" url={playgroundLogo} />
+        </Flex>
+
+        <FlexItem justify="end">
+          <Button marginBottom="xs" marginRight="lg" text="Share" />
```

Comment:
```suggestion
          <Button disabled marginBottom="xs" marginRight="lg" text="Share" />
```

I know the story doesn't call for it, but because this button doesn't do anything yet. Making it disabled would be fun. 

Don't do this if you don't want to.